examples/micro-ros_publisher_wifi/micro-ros_publisher_wifi.ino

```diff
@@ -18,7 +18,11 @@ rclc_support_t support;
 rcl_allocator_t allocator;
 rcl_node_t node;
 
-#define LED_PIN 13
+#ifdef ARDUINO_GIGA
```

Comment:
I'd change this ifdef into
```patch
#if defined (LED_BUILTIN)
#define LED_PIN LED_BUILTIN
#else
#define LED_PIN 13
#endif
```
for future proofness :slightly_smiling_face:

Reply:
@facchinm done! thanks for your suggestion